lib/debezium/converters/time.go

```diff
@@ -12,7 +12,7 @@ import (
 type Time struct{}
 
 func (Time) ToKindDetails() typing.KindDetails {
-	return typing.NewKindDetailsFromTemplate(typing.ETime, ext.TimeKindType)
+	return typing.NewExtendedTimeDetails(typing.ETime, ext.TimeKindType, "")
```

Comment:
Should we make this method take a receiver `t` and use `t.layout()` as the last arg here?